src/libraries/System.IO.FileSystem/tests/FileStream/CharacterDevice.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    [PlatformSpecific(TestPlatforms.AnyUnix)]
+    public class CharacterDevice
+    {
```

Comment:
Maybe we also want a test using some special files against the File.Write* APIs that also use RandomAccess internally.

Reply:
Yes, I think it would be good to preserve the state in test -- what is or isn't supported -- of known file types. This way, future regressions of intentional changes won't go unnoticed.

> fifo (S_IFIFO), character device (S_IFCHR), block device (S_IFBLK),
socket (S_IFMT), sticky bit (S_ISVTX), directory (S_IFDIR),
symlink (S_IFLNK) and the regular file (S_IFREG)

Reply:
Done.